libr/bin/p/bin_elf.inc

```diff
@@ -569,6 +569,9 @@ static RList* libs(RBinFile *bf) {
 
 static RBinReloc *reloc_convert(struct Elf_(r_bin_elf_obj_t) *bin, RBinElfReloc *rel, ut64 GOT) {
 	r_return_val_if_fail (bin && rel, NULL);
+	if (GOT == UT64_MAX) {
```

Comment:
if got is -1 i think it will be safer to return null

Reply:
yes probably... however, as said somewhere else (i'm getting every time more confused about where you are implementing what... please just create one pr with all the changes, because they seem too much related) if GOT is -1 it means there is some code that is putting that invalid value in the reloc_list. that code should be fixed to prevent that insertion, if possible.

Reply:
the ELF PR got splitted into 5 different ones, each one with specific changes. this one have only the remaining changes only related to ut64mas as invalid. ill take care of them during this week